pkg/logger/log_events.go

```diff
@@ -0,0 +1,52 @@
+package logger
```

Comment:
Why is this a separate file?

Reply:
because it doesn't make sense to write tests for the content of the file (it is ignored in the makefile)

Reply:
Why not just ignore the original file `logger.go`?

Reply:
because some of the logger context needs to be tested (see the logger_test.go). The rest doesn't return any value and doesn't make sense to test it

Reply:
Yes, but we can still have the test and ignore the file for code coverage.  

I saying that because I am not convinced by the split, as the design/code separation is then driven by if something is testable or not to be added to the code coverage.

Reply:
ok. I can change it

Reply:
@machi1990 - done

Reply:
Thanks @pawelpaszki